src/runtime_src/core/tools/xbutil2/SubCmdValidate.cpp

```diff
@@ -1674,8 +1710,14 @@ SubCmdValidate::execute(const SubCmdOptions& _options) const
   // Collect all of the tests of interests
   std::vector<TestCollection *> testObjectsToRun;
 
+  /*
+   * Iterate through the test suites and compare them against the desired user tests
+   * If a match is found enqueue the test suite to be executed
+   */
   for (unsigned index = 0; index < testSuite.size(); ++index) {
```

Comment:
Not your change, but unsigned what?   Plus testSuite.size() problem returns some type integral that doesn't match with int, and any static code analyzer would complain.